docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-vpc-intro.md

```diff
@@ -81,13 +81,46 @@ The block of [private IP addresses](#private-ip-address-ranges) used to define y
 
-When entering the range for your VPC in YugabyteDB Managed, the size of the network is determined by the prefix length (the number after the `/`). YugabyteDB Managed supports network sizes from `/26` to `/16` as shown in the following table. For typical applications, `/26` is more than sufficient.
+When entering the range for your VPC in YugabyteDB Managed, the size of the network is determined by the prefix length (the number after the `/`). YugabyteDB Managed supports network sizes from `/26` to `/16`. For typical applications, `/25` is sufficient.
 
-| Provider | Network Size (prefix length) | Number of Usable IP Addresses | Notes |
+The number of available addresses and sizing recommendation depends on the cloud provider where you are deploying.
+
+{{< tabpane text=true >}}
+
+  {{% tab header="AWS" lang="aws" %}}
+
+In AWS, you assign the range to a single region. If you need multiple regions, you create a separate VPC for each region.
+
+Use at least `/25` for production deployments, and `/24` if deploying multiple clusters in a single VPC. `/26` should only be used for testing and development.
+
+When sizing the VPC, you need to take into account that the address range is split into subnets, each in a separate availability zone, and that AWS reserves 5 addresses per subnet. A further 8 addresses are required by AWS when creating the load balancer (though typically only one or two addresses are used while running). If you enable public access on your cluster, then two load balancers are created, one for VPC peered connections, and one for public connections.
+
+For example, in a size `/26` VPC, each subnet is size `/28`, which is 16 addresses per subnet; 5 addresses are reserved by AWS, and 8 addresses are required to create a load balancer, which leaves only 3 usable addresses. This limits you to 3 nodes per zone in a `/26` VPC with the regular load balancer, and only 2 nodes if you want to enable public access.
+
+| Network Size<br/>(prefix length) | IP Addresses | IP addresses per subnet | Available IP addresses per subnet |
+| :--- | :--- | :--- | :--- |
+| /26<br/>/25<br/>/24 | 64<br/>128<br/>256 | 16<br/>32<br/>64 | 2-3<br/>18-19<br/>50-51 |
+
+For more information, refer to [Subnets for your VPC](https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html) in the AWS documentation.
+
+  {{% /tab %}}
+
+  {{% tab header="GCP" lang="gcp" %}}
+
+For a custom GCP network, a size of /26 per region is sufficient for typical applications.
+
+For an automatic GCP network, a minimum size of /18 is recommended to have enough addresses to distribute among all the regions.
```

Comment:
(Formatting to be consistent with the AWS tab. You could also go with no code font for either, as you've done in the tables.)
```suggestion
For a custom GCP network, a size of `/26` per region is sufficient for typical applications.

For an automatic GCP network, a minimum size of `/18` is recommended to have enough addresses to distribute among all the regions.
```